product_quantity_update_force_inventory/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,11 @@
+With this module, when a user goes to a product and presses the button
+'Update Quantity' they will be directed to a new Inventory Adjustment
+for this product, instead of driving the user to update the quantities
+in the stock quants.
+
+By doing this users will be able to have full traceability on the
```

Comment:
I'm wondering what is the difference with standard flow as:

- If you update the quantity through quants, you'll get the move name with 'Quantity updated'-like message
- If you use the Action, you can add a reason:
![image](https://github.com/OCA/stock-logistics-warehouse/assets/19529533/cbbc1eca-53cc-4b3b-8f30-0329fe7f1425)


![image](https://github.com/OCA/stock-logistics-warehouse/assets/19529533/a82ed1e0-d8fc-46a9-b996-d8907fafc1b2)

Reply:
As it is stated in this pull request https://github.com/OCA/stock-logistics-warehouse/pull/846, this module was created to force users to create inventory adjustments instead of directly updating quantities on the product card but since the stock.inventory is no longer available, now it is not possible to start inventory adjustments without updating quantities. Given these changes, I adapted the module to redirect users to the inventory adjustment view filtered by the product when they initiate a quantity update. However, I recognize that this adaptation does not fully replicate the original functionality of the module.
Since the module has lost its original purpose, what do you think it would be appropiate to do with it?

Reply:
Yes, I know the drawback of not using inventory lines anymore.

But, IMHO, we should try not duplicating existing functionalities as it can add confusion for users, add an additional entry point to do the same thing.

I would say, something has to be done on the actions visible on stock quant view (in inventory mode):

![image](https://github.com/OCA/stock-logistics-warehouse/assets/19529533/97fae593-4ce1-4377-933f-22b5afdb6851)


For that 'Apply' action, something should be done to display the reason popup too if needed.

Reply:
In fact your functionality is not needed anymore. Odoo has managed that well. If you don't have 'Storage Locations' enabled on settings, the popup will be launched as there is only 1 location (Stock). That popup is launched if you click on the magic button 'On Hand' in the product form.

If you enable 'Storage Locations' setting, you are both redirected to the stock quant view.

Reply:
Yes, i agree with you. I will be closing this PR as it is not needed anymore.
Thank you for your cooperation.

Reply:
@AnaisLForgeFlow The only missing feature is to be able to add a reason for one line (the Apply button on the line)

Reply:
But the feature that you mean is what is being added in the stock_change_qty_reason module, right?